entities-search/src/test/scala/io/renku/entities/search/WorkflowsEntitiesFinderSpec.scala

```diff
@@ -184,4 +185,19 @@ class WorkflowsEntitiesFinderSpec
       wfs.map(_.workflowType) should contain theSameElementsAs List(WorkflowType.Step, WorkflowType.Composite)
     }
   }
+
```

Comment:
Shouldn't the new test go to a new `ProjectEntitiesFinderSpec`?

Reply:
Oh, right 🤦🏼 - I think I wanted it in `EntityFindersSpec`

Reply:
Yes, that's also good.